src/main/java/org/apache/flink/benchmark/full/SerializationFrameworkAllBenchmarks.java

```diff
@@ -85,4 +90,76 @@ public void serializerAvroReflect(FlinkEnvironmentContext context) throws Except
 
 		env.execute();
 	}
+
+	@Benchmark
+	@OperationsPerInvocation(value = SerializationFrameworkMiniBenchmarks.RECORDS_PER_INVOCATION)
+	public void serializerKryoThriftWithoutRegistration(FlinkEnvironmentContext context) throws Exception {
+		StreamExecutionEnvironment env = context.env;
+		env.setParallelism(4);
+		ExecutionConfig executionConfig = env.getConfig();
+		executionConfig.enableForceKryo();
+		executionConfig.addDefaultKryoSerializer(org.apache.flink.benchmark.avro.MyPojo.class, TBaseSerializer.class);
+		executionConfig.addDefaultKryoSerializer(org.apache.flink.benchmark.avro.MyOperation.class, TBaseSerializer.class);
+
+		env.addSource(new ThriftPojoSource(RECORDS_PER_INVOCATION, 10))
+				.rebalance()
+				.addSink(new DiscardingSink<>());
+
+		env.execute();
+	}
+
+	@Benchmark
+	@OperationsPerInvocation(value = SerializationFrameworkMiniBenchmarks.RECORDS_PER_INVOCATION)
+	public void serializerKryoThrift(FlinkEnvironmentContext context) throws Exception {
```

Comment:
You have put the benchmarks in `AllBenchmarks`, that are not executed automatically contrary to the `SerializationFrameworkMiniBenchmarks`. Was it intended?

Reply:
yes - I expect no changes in these benchmarks due to Flink code changes since they all go through Kryo.